spec/models/bundle_context_spec.rb

```diff
@@ -37,6 +37,16 @@
         expect { bc.project_name = valid_chars }.not_to change(bc, :valid?).from(true)
       end
     end
+
+    context 'bundle_dir must be a sub dir of allowed parent directories' do
+      it 'sub dir of parent directories' do
+        expect { bc.bundle_dir = 'tmp/' }.not_to change(bc, :valid?).from(true)
+      end
+
+      it 'not a sub dir of parent directories' do
+        expect { bc.bundle_dir = 'spec/../../' }.to change(bc, :valid?).to(false)
```

Comment:
what about using `'spec/test_data/../../'` instead?  that way, the test path's first few segments would be valid were it not for the `..` (and thus more plausible as an attack if we were being more naive about our path normalization and comparison).